### PR TITLE
chore: simplify comparison and variable declaration

### PR DESCRIPTION
**Pull request by hezhizhen**
### Ⅰ. Describe what this PR does

Simplify some code, mainly comparison and variable declaration.

### Ⅲ. Describe how to verify it

lint and test

### CODE REVIEW

File: pkg/controller/broadcastjob/broadcastjob_controller.go [hunk 1]
```diff
@@ -178,7 +175,8 @@ func (r *ReconcileBroadcastJob) Reconcile(_ context.Context, request reconcile.R
 	addLabelToPodTemplate(job)
 
 	if IsJobFinished(job) {
-		if isPast, leftTime := pastTTLDeadline(job); isPast {
+		isPast, leftTime := pastTTLDeadline(job)
```
Comment (zmberg) [inline pkg/controller/broadcastjob/broadcastjob_controller.go:178]: Why you need to change here?

Comment (hezhizhen) [reply pkg/controller/broadcastjob/broadcastjob_controller.go:178]: I wanted to write something like 

```go
if !isPast && leftTime>0 { return xxx }

klog.Infof(xxx)
...
return xxx
```

But I was not sure of its correctness, so I reverted it, but this modification was kept.

**Pull request merged**